packages/broker/src/ConfigWizard.ts

```diff
@@ -164,12 +171,17 @@ export const getConfigFromAnswers = (answers: any): Config => {
     pluginNames.forEach((pluginName) => {
         const template = PLUGIN_TEMPLATES[pluginName]
         if (answers.selectPlugins && answers.selectPlugins.includes(pluginName)){
-            // the publishHttp plugin is special, it needs to be added to the config after the other plugins
-            if (pluginName === 'publishHttp') {
+            if (answers[`${pluginName}Port`] === template.port){
+                // user selected default value, enable without specifying the port
+                config.plugins![pluginName] = {}
```

Comment:
If we modify `config.plugins![pluginName]` in each section, maybe that could be outside the `if`s

E.g.
```
let pluginConfig = {}
if ... {
...
}
config.plugins![pluginName] = pluginConfig
```